Src/StartMenu/StartMenuDLL/StartMenuDLL.cpp

```diff
@@ -2956,6 +2969,12 @@ static void InitStartMenuDLL( void )
 	if (GetSettingBool(L"CustomTaskbar"))
 	{
 		auto module=GetModuleHandle(L"taskbar.dll");
+		if (!module)
+		{
+			module = GetModuleHandle(L"ep_taskbar.5.dll");
```

Comment:
Perhaps I should come a up with a way so that any program can read which ep_taskbar is active... hardcoding the module name is not a good idea.

There are currently 6 variants of the taskbar, of which variants 0 (10 VB), 2 (11 NI), and 5 (11 GE) are under active use. Hardcoding this to 5 would make it only target the taskbar if it's running under 11 GE/24H2. 

https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation#supported-windows-versions

Reply:
Thanks.

I have added check also for version `2` as it seems `2` an `5` are the only officially supported by `ExporerPatcher`.
I guess that should be enough.

I have also added some more safety checks to `ExtTextOutW` hook.